overview.html

```diff
@@ -0,0 +1,452 @@
+<h2 id="Cookie authentication">Cookie authentication</h2>
+
+<P>
+    Using the {@link com.cloudant.client.api.ClientBuilder#username(String)} and
+    {@link com.cloudant.client.api.ClientBuilder#password(String)}
+    options uses <a href="https://docs.cloudant.com/authentication.html#cookie-authentication">
+    cookie authentication</a> for the CloudantClient connection. The supplied credentials are
+    used
+    to request a session with the server and the session is renewed automatically if the cookie
+    expires. If the credentials become invalid then a new instance of a CloudantClient needs to
+    be
+    created.
+</P>
+
+<h2 id="Custom authentication">Custom authentication</h2>
+
+<P>
+    If cookie authentication is not desired then it is possible to build the CloudantClient
+    without
+    credentials and use a {@link HttpConnectionInterceptor} to customize the HTTP request with
+    the
+    desired authentication mechanism.
+</P>
+
+<P>
+    An example using Basic Authentication is shown in the javadoc for
+    {@link com.cloudant.client.api.ClientBuilder#interceptors(HttpConnectionInterceptor[])}.
+</P>
+
+<h1 id="Functions">Functions</h1>
+
+<h2 id="Server Functions">Server Functions</h2>
+
+<P>
+    {@link com.cloudant.client.api.CloudantClient} encapsulates the connection to the server and
+    provides access to server functions.
+</P>
+
+<h2 id="Database Functions">Database Functions</h2>
+
+<P>
+    {@link com.cloudant.client.api.CloudantClient#database(String, boolean)} returns a
+    {@link com.cloudant.client.api.Database} object that provides access to database functions.
+</P>
+
+<h2 id="Document Functions">Document Functions</h2>
+
+<h3 id="Single document CRUD">Single document CRUD</h3>
+
+<P>
+    CRUD operations for documents in a database are achieved via the
+    {@link com.cloudant.client.api.Database} object.
+</P>
+<UL>
+    <LI>Create - {@link com.cloudant.client.api.Database#save(Object)} or {@link
+        com.cloudant.client.api.Database#post(Object)}
+    </LI>
+    <LI>Read - {@link com.cloudant.client.api.Database#find(Class, String)}</LI>
+    <LI>Update - {@link com.cloudant.client.api.Database#save(Object)} or {@link
+        com.cloudant.client.api.Database#update(Object)}
+    </LI>
+    <LI>Delete - {@link com.cloudant.client.api.Database#remove(Object)}</LI>
+</UL>
+
+<h3 id="Bulk document CRU">Bulk document CRU</h3>
+
+<P>
+    {@link com.cloudant.client.api.Database} also provides a bulk documents API for fetching or
+    modifying multiple documents in a single request.
+</P>
+<UL>
+    <LI>Create/Update - {@link com.cloudant.client.api.Database#bulk(List)}</LI>
+    <LI>Read - {@link com.cloudant.client.api.views.AllDocsRequestBuilder} via {@link
+        com.cloudant.client.api.Database#getAllDocsRequestBuilder()}
+    </LI>
+</UL>
+
+<h2 id="Attachment Functions">Attachment Functions</h2>
+
+See the <a href="https://docs.cloudant.com/attachments.html">Cloudant documentation</a> for more
```

Comment:
`href target="_blank"` to open in a new window? In the three pane view this currently loads the content in the main pane which doesn't look right